projects/solar_master/src/solar_master_slave.c

```diff
@@ -0,0 +1,38 @@
+#include "solar_master_slave.h"
+
+static void prv_voltage_reading_cb(uint32_t value, void *context) {
+  SolarMasterSlave *slave = context;
+
+  slave->sliding_sum_voltage -= slave->averaging_voltage[slave->counter];
+  slave->sliding_sum_temp -= slave->averaging_temp[slave->counter];
+  slave->averaging_voltage[slave->counter] = (int32_t)value >> 16;
+  slave->averaging_temp[slave->counter] = (int32_t)(value && ((1 << 16) - 1));
+  slave->sliding_sum_voltage += slave->averaging_voltage[slave->counter];
+  slave->sliding_sum_temp += slave->averaging_temp[slave->counter];
+  printf("WEE\n");
```

Comment:
What does this mean? 🤔

Reply:
My bad, I need to put that in a CAN message